external/fv3fit/fv3fit/sklearn/_random_forest.py

```diff
@@ -356,40 +297,44 @@ def load(cls, path: str) -> "SklearnWrapper":
         packer_config_dict = metadata.get("packer_config", {})
         packer_config = dacite.from_dict(PackerConfig, packer_config_dict)
 
-        obj = cls(input_variables, output_variables, model, packer_config=packer_config)
+        obj = cls(
+            input_variables,
+            output_variables,
+            regressor,
+            n_jobs,
+            packer_config=packer_config,
+        )
         obj.target_scaler = scaler_obj
         output_features_dict = metadata["output_features"]
-        if isinstance(output_features_dict, dict):
-            output_features_ = dacite.from_dict(PackingInfo, data=output_features_dict)
-        else:
-            # older model format, uses tuple instead
-            output_features_ = PackingInfo.from_tuples(output_features_dict[1])
+        output_features_ = dacite.from_dict(PackingInfo, data=output_features_dict)
         obj.output_features_ = output_features_
 
         return obj
 
-    @property
-    def feature_importances(self) -> Sequence[float]:
-        importances = []
-        for member in self.model.regressors:
-            importances.append(member.feature_importances_)
-        return importances
-
-    @property
-    def mean_importances(self) -> np.ndarray:
-        return np.array(self.feature_importances).mean(axis=0)
-
-    @property
-    def std_importances(self) -> np.ndarray:
-        return np.array(self.feature_importances).std(axis=0)
+    @staticmethod
+    def _load_regressor(b: bytes) -> Tuple[sklearn.base.BaseEstimator, int]:
+        regressor_components = joblib.load(io.BytesIO(b))
+        if isinstance(regressor_components["regressors"], list):
+            # backward compatibility for previous models saved as single batch regressor
+            if len(regressor_components["regressors"]) == 1:
+                regressor = regressor_components["regressors"][0]
+            else:
+                raise ValueError(
+                    "Cannot load older models that saved multiple batch regressors."
+                )
+        else:
+            regressor = regressor_components["regressors"]
+        return regressor, regressor_components["n_jobs"]
 
     def input_sensitivity(self, stacked_sample: xr.Dataset) -> InputSensitivity:
         _, input_multiindex = pack(
             stacked_sample[self.input_variables], ["sample"], self.packer_config
         )
         feature_importances = {}
         for (name, feature_index), mean_importance, std_importance in zip(
-            input_multiindex, self.mean_importances, self.std_importances
+            input_multiindex,
```

Comment:
The rest of the `SklearnWrapper` class now works for generic sklearn estimators, with the exception of this method. Could you make a child `SklearnWrapperRF` class that inherits from `SklearnWrapper` and has this method for itself?

Reply:
As discussed will move this to a method of the RandomForest rather than of the SklearnWrapper, which should achieve the same goal of having the SklearnWrapper work for any sklearn estimator.